core/src/test/java/org/pentaho/di/core/row/value/ValueMetaBaseTest.java

```diff
@@ -700,16 +704,6 @@ public void testSetPreparedStatementStringValueDontLogTruncated() throws KettleD
     assertEquals( 0, events.size() );
   }
 
-  @Test
```

Comment:
This test is redundant. Included the logging validation in `test_Pdi_17126_postgres_truncate()`.